Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+WORKDIR /clang-bind
+COPY . .
+
+RUN apt-get update && apt-get install -y python3-pip build-essential wget
+
+# Install libclang via development branch on https://apt.llvm.org/
+RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+RUN echo 'deb http://apt.llvm.org/focal/ llvm-toolchain-focal-12 main' | tee -a /etc/apt/sources.list
+RUN echo 'deb-src http://apt.llvm.org/focal/ llvm-toolchain-focal-12 main' | tee -a /etc/apt/sources.list
+RUN apt-get install -y libclang-12-dev python3-clang-12
```

Comment:
Collapse into a single RUN?

Reply:
Let's keep libclang installation separate:
https://github.com/PointCloudLibrary/clang-bind/blob/54d7879275e6e5392aa13c56345583b7817ced8b/Dockerfile#L6-L12

Reply:
Sure, we can do that. BTW, should we run an apt update after modifying the sources?

Reply:
Yes, we should. However, not need in the new changes.